crates/ethportal-api/src/types/execution/header.rs

```diff
@@ -309,6 +313,63 @@ impl PartialEq for Header {
     }
 }
 
+impl From<ExecutionPayloadBellatrix> for Header {
```

Comment:
This logic is somewhat duplicated from https://github.com/ethereum/trin/blob/master/bin/trin-execution/src/era/beacon.rs#L39 but I wasn't able to find a great way to use integrate the two use-cases together

Reply:
I think this level of code duplication is fine, but I do think we should get this PR in first https://github.com/ethereum/trin/pull/1721